File: C++/factorial.cpp
// // factorial of a given number

#include<iostream>
using namespace std;


int fact(int n){
    int factorial=1;
    for ( int i = 2; i <=n; i++)
    {
    factorial = factorial*i;
    }
    return factorial;
}

int main(){
    int n;
    cin>>n;
     fact(n);
     cout<<fact(n);
    return 0;
}

File: Factorial.java
importjava.util.*;
public classProgram2
{
public static voidmain(String[] args)
{
Scanner scn=newScanner(System.in);
intfact =1;
System.out.print("Enter a number: ");
intn=scn.nextInt();
for(inti =1; i <=n; i++)
{
fact *= i;
}
System.out.println("Factorial of "+n+" is "+ fact);
}
}
